### PR TITLE
[Fix] New configuration for hw1-config.

### PR DESCRIPTION
**Pull request by paul-amonson**
Signed-off-by: Paul Amonson <paul.d.amonson@intel.com>

### CODE REVIEW

File: docker-deploy/hw1-config/cli_config.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "rest_server_address" : "localhost",
+  "rest_server_address" : "am01-nmn.local",
```
Comment (sindhu-106) [inline docker-deploy/hw1-config/cli_config.json:2]: If adapters are trying to connect to "api-gw-service-nmn.local" shouldnt this be "api-gw-service-nmn.local" ?

File: docker-deploy/hw1-config/MachineConfig.json [hunk 2]
```diff
@@ -13,84 +13,84 @@
       "Key": "UcsConfigDirectory"
     },
     {
-      "Value": "sms01-nmn",
+      "Value": "am01-nmn.local",
```
Comment (sindhu-106) [inline docker-deploy/hw1-config/MachineConfig.json:16]: If adapters are trying to connect to "api-gw-service-nmn.local" shouldnt this be "api-gw-service-nmn.local" ?

File: docker-deploy/hw1-config/cli_config.json (hunk 1 above)
Comment (paul-amonson) [reply docker-deploy/hw1-config/cli_config.json:2]: No the cli talks to the UI adapter not the underlying hw apis.

File: docker-deploy/hw1-config/MachineConfig.json (hunk 2 above)
Comment (paul-amonson) [reply docker-deploy/hw1-config/MachineConfig.json:16]: No the cli talks to the UI adapter not the underlying hw apis.

**Pull request merged**